lib/mayaUsd/render/pxrUsdMayaGL/shapeAdapter.h

```diff
@@ -201,14 +201,15 @@ class PxrMayaHdShapeAdapter
 
         /// Get whether this shape adapter is for use with Viewport 2.0.
         ///
-        /// The shape adapter gets its viewport renderer affiliation from the
-        /// version of Sync() that is used to populate it.
+        /// The shape adapter's viewport renderer affiliation must be specified
+        /// when it is constructed.
         ///
         /// Returns true if the shape adapter should be used for batched
         /// drawing/selection in Viewport 2.0, or false if it should be used
         /// in the legacy viewport.
-        MAYAUSD_CORE_PUBLIC
-        virtual bool IsViewport2() const;
+        bool IsViewport2() const {
```

Comment:
Just to be sure, we don't need to export this method?

Reply:
Right, we don't need to export this anymore since it's now defined inline right here.